src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -9373,19 +9166,15 @@ private static void payActionCost(final String responseText) {
         }
         break;
 
-      case SkillPool.MICROMETEOR:
-        Preferences.increment("_micrometeoriteUses");
```

Comment:
this has disappeared with no apparent replacement. While you can use it an unlimited number of times per day, the % delevel decreases with use.

Reply:
You know what, for years I had thought this pref was added by mistake and had always meant to remove it. TIL!